### ISSUE TITLE
Name of directory should not be hard coded

### ISSUE THREAD

**Issue by hartenthaler** [08ah15/WeEt#29]
I tried this custom module in a directory named WeEt but that does not work. The directory name has to be WeEt-main. That isn't good.

**Comment by 08ah15** [comment]
Agreed! Any hint is welcome. At the time I did my last programming Assembler, Basic and Pascal were in. First time on Git. I have two branches. That is why „main“ appears. However, I did not figure if this is a matter of a setting or just because I have my playground here, too. if the later is the case how is a playground best set up when working from multiple machines?
